.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchDateParserConfiguration.cs

```diff
@@ -136,7 +139,7 @@ public DutchDateParserConfiguration(ICommonDateTimeParserConfiguration config)
 
         public IDateTimeUtilityConfiguration UtilityConfiguration { get; }
 
-        public int GetSwiftMonth(string text)
+        public int GetSwiftMonthOrYear(string text)
```

Comment:
This is a rename for consistency with parsers in other cultures?

Reply:
Yes, the method could also be reused in Year, so i rename the method.